Add iometer integration documentation

jukrebs opened the pull request:
<!--
  You are amazing! Thanks for contributing to our project!
  Please, DO NOT DELETE ANY TEXT from this template! (unless instructed).

  Before submitting your pull request, please verify that you have chosen the correct target branch,
  and that the PR preview looks fine and does not include unrelated changes.
-->
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Add documentation for IOmeter integration


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/135513
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/6348
- This PR fixes or closes issue: fixes #

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## Summary by CodeRabbit

- **Documentation**
	- Added comprehensive documentation for the IOmeter integration.
	- Introduced configuration details and supported sensor metrics.
	- Specified integration requirements and compatibility information.

- **New Features**
	- Integrated IOmeter device support in Home Assistant.
	- Enabled local polling for device data collection.
	- Supported configuration through config flow.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Commit:
add iometer integration documentation

frenck reviewed (dismissed):
Hi there @MaestroOnICe 👋 

I could not find the pull request in the core repository that adds this new integration. Could you point me to it?

../Frenck

jukrebs committed:
Update iometer.markdown

MaestroOnICe commented:
> Hi there @MaestroOnICe 👋
> 
> I could not find the pull request in the core repository that adds this new integration. Could you point me to it?
> 
> ../Frenck

Hi Frenck,

I also updated this PR to be linked to the core repository PR. 

Regards

jukrebs committed:
remove Core/Bride status from doc

Remove Core/Bridge status for now since it is supposed to be a binary sensor and will be added in the future.
